pkg/controller/propagator/propagation.go

```diff
@@ -210,6 +201,89 @@ func (r *ReconcilePolicy) handleRootPolicy(instance *policiesv1.Policy) error {
 	return nil
 }
 
+// getApplicationPlacementDecisions return the placement decisions from an application
+// lifecycle placementrule
+func getApplicationPlacementDecisions(c client.Client, pb policiesv1.PlacementBinding, instance *policiesv1.Policy) ([]appsv1.PlacementDecision, *policiesv1.Placement, error) {
+	plr := &appsv1.PlacementRule{}
+	err := c.Get(context.TODO(), types.NamespacedName{Namespace: instance.GetNamespace(),
+		Name: pb.PlacementRef.Name}, plr)
+	if err != nil && !errors.IsNotFound(err) {
```

Comment:
Did you decide on how to handle the case when the error is not found? For instance, using `plr.GetName()` doesn't seem right if `plr` is just `plr := &appsv1.PlacementRule{}`.

Reply:
I accidentally created all updated to propagation.go in an editor opened for a different directory so the changes were not really there when I requested a new review.  The changes are there now.